### ISSUE TITLE
Broken Pipe Issue every 5-15 Minutes

### ISSUE THREAD

**Issue by rselldorf** [iterate-ch/cyberduck#14969]
Version 8.6.2 (40032) | Ventura 13.5 | Broken Pipe Issue every 5-15 Minutes

Regards
Robert

**Comment by Cristal-Solutions** [comment]
Hello,
Same problem here since the last update. Is there a setting to change ?
Thank you very much.
Best regards

**Comment by ylangisc** [comment]
We need more context. Can you please provide a [debug log](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level) showing the issue?

**Comment by rselldorf** [comment]
Anhang verfügbar bis 06.09.2023
thanks for the quick response
Zum Laden klicken
 <https://www.icloud.com/attachment/?u=https%3A%2F%2Fcvws.icloud-content.com%2FB%2FAYFMS1groO0Z1fR_2ncOJsDmGm2SAYGV6kq64M6cQN3j9KIiEsoV6IBI%2F%24%7Bf%7D%3Fo%3DAs4Ex2bv5Gd2RSa2XAhyI0C3lAcy_2ddrzTPvQv6pv7a%26v%3D1%26x%3D3%26a%3DCAogY6iy9eKPGNGyuLH3Rbat9qh8Q02QIdyXWcIwrwn2wZYSdhDkh8_-nDEY5JfK0qYxIgEAKgkC6AMA_1bHBPZSBOYabZJaBBXogEhqJUUizl7Re9uVnwpeVN6W7_lwEHwumJA6bFi7ZWkOIEOPg26RJdZyJQILpQpMLDS2ehHHr1nlA4AoQB8BHdZdaTVJPmeDcdUWGo9VVn8%26e%3D1694000909%26fl%3D%26r%3D6BBA032A-7A4E-4C7F-97A3-BF1A67B52CDA-1%26k%3D%24%7Buk%7D%26ckc%3Dcom.apple.largeattachment%26ckz%3D8008543F-4B88-41C2-A18C-10BA4865B172%26p%3D125%26s%3DHRB5kZ2M6G5SRce1PGT7xsNLI8A&uk=dcsXSfCqulOd-k61xGwLNw&f=cyberduck.log&sz=52873650>cyberduck.log
52,9 MB



Robert Selldorf
***@***.***
T: +49 1607124386

**Comment by Cristal-Solutions** [comment]
Hello, 
Here is my log : [cyberduck.log](https://github.com/iterate-ch/cyberduck/files/12279471/cyberduck.log)
Thank you very much.

**Comment by bradyg77** [comment]
Hello - Same issue on Mac OS 13.4.1, Ever since latest CyberDuck update (Version 8.6.2 (40032)).  Broken Pipe errors every few minutes, even when idle with no transfers - then cannot initiate any new transfers, create folders, etc. after errors begin - must close connection and re-connect to server, then all is OK for a few minutes again (FTP running on unraid - no recent changes to server or network environment).  If broken pipe errors begin appearing during active transfers, the active transfers are not impacted and still complete successfully.
<img width="263" alt="Screenshot 2023-08-07 at 1 22 40 PM" src="https://github.com/iterate-ch/cyberduck/assets/88783583/c45c4fa5-f9be-4b31-9b87-54d4c2858ce5">

**Comment by ylangisc** [comment]
Thanks for the log files. Is there any chance to get a test account to be able to reproduce the issue as I don't see any reason in the logs? Feel free to send it to support@cyberduck.io for privacy reasons.

**Comment by mprove** [comment]
same here. Mountain Duck has the same issue. Is there a way to downgrade? 
My log from CyberDuck:
[mprove-cyberduck.log](https://github.com/iterate-ch/cyberduck/files/12300963/mprove-cyberduck.log)

CyberDuck 8.6.2 / macOS 13.5 
Mountain Duck 4.14.2

**Comment by rselldorf** [comment]
I am sorry, but this will not be possible.
Would Teamviewer be of any help?

Can I revert to an old version?

Robert



Robert Selldorf
***@***.***
T: +49 1607124386

**Comment by ylangisc** [comment]
Thanks for providing details and logs. In the meantime I'm able to reproduce the issue and we are working on a fix.

**Comment by dkocher** [comment]
Regression from 05a029b5.

**Comment by dkocher** [comment]
Version _[8.6.3](https://trac.cyberduck.io/milestone/8.6.3)_ with a fix has now been released.

**Comment by porg** [comment]
I often had this broken pipe issues, also before this version. But a change of user behavior for me fixed it.

1. Open Cyberduck
2. Edit file with TextEditorOfChoice (creates file within  `/tmp/*` dir)
3. Mac goes to sleep. Mac wakes some time later.
4. First go to Cyberduck and perform a refresh action. May take 20secs or so. In doubt create a blank file, go elsewhere, go back, delete it. Then you certainly now the pipe is upright.
5. Only then resume in your TextEditorOfChoice.
 
If your first action after Mac wake is to go to the TextEditor perform some changes and save, then CyberDuck always got the beachball. Sometimes managed to continue after 2-3min, but often I had to force crash. Since I changed my order of operation after the wake event I had no crashes ever.
